common/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
+    <string name="staking_rewards_event">Event</string>
+    <string name="common_copy_event_id">Copy Event Id</string>
+    <string name="staking_slash_details">Slash details</string>
```

Comment:
We can just use "Details" title for all details screen (Transfer/Reward & Slash /Extrinsic)